infra-as-code/bicep/modules/policy/definitions/mc-custom-policy-definitions.bicep

```diff
@@ -3,6 +3,9 @@ targetScope = 'managementGroup'
 @description('The management group scope to which the policy definitions are to be created at. DEFAULT VALUE = "alz"')
 param parTargetManagementGroupID string = 'alz'
 
+@description('Set Parameter to true to Opt-out of deployment telemetry')
```

Comment:
As for custom role defintions, you probably want to hold off on this.

Reply:
this has been renamed to mc-customPolicyDefinitions.bicep, and I will get the new output from workflow script because there have been changes from the ES upstream repository